06.understand_sentiment/README.md

```diff
@@ -163,9 +163,9 @@ def stacked_lstm_net(input_dim,
                      stacked_num=3):
     """
     A Wrapper for sentiment classification task.
-    This network uses bi-directional recurrent network,
-    consisting three LSTM layers. This configure is referred to
-    the paper as following url, but use fewer layrs.
+    This network uses a bi-directional recurrent network,
+    consisting three LSTM layers. This configure refers to
```

Comment:
`This configuration is motivated from the following paper, but uses few layers`.
Also make the paper a hyperlink and remove the link from the text.